packages/start-frontend/__tests__/cli.test.ts

```diff
@@ -1,168 +1,190 @@
-      const results = await exeInteractive(cli, [
-        testDir,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-        keys.ENTER,
-      ]);
+  function nextPrompt(inputs: string[]) {
+    if (!inputs.length) return;
 
-      expect(results).toContain(`start-frontend`);
-      expect(results).toContain(`Welcome!`);
-      expect(results).toContain(
-        `? Where Would You like to Create Your Application?`
-      );
-      expect(results).toContain(`? Select a JavsScript library for UI`);
-      expect(results).toContain(`? Select an API Solution`);
-      expect(results).toContain(`? Select module do you want to use`);
-      expect(results).toContain(`? Add Testing codes for Catching bugs early?`);
-      expect(results).toContain(`? Add Vitest for Unit Testing?`);
-      expect(results).toContain(`? Add Storybook for Visual Testing?`);
-      expect(results).toContain(`? Add Playwright for End-To-End Testing?`);
-      expect(results).toContain(`? Add ESLint for Code Linting?`);
-      expect(results).toContain(`? Add Prettier for Code Formatting?`);
-      expect(results).toContain(`Success! Created a new app at "my-test".`);
-    });
-  });
+    // Write the input to the CLI process with a delay
+    setTimeout(() => {
+      cliProcess?.stdin?.write(inputs[0]);
+      nextPrompt(inputs.slice(1));
+    }, delay);
+  }
 
-  // TODO: Skip testing as it is not yet implemented.
-  describe.skip("install react boilerplate to specify dir", () => {
-    test("install", async () => {
-      await exe("node", [startFrontend, testDir], { cwd });
-
-      const expectDirs = [
-        ".env.template",
-        ".eslintrc.js",
-        ".gitignore",
-        "README.md",
-        "__mocks__",
-        "babel.config.js",
-        "index.html",
-        "jest-setup.ts",
-        "package.json",
-        "public",
-        "src",
-        "tests",
-        "tsconfig.json",
-        "vite.config.ts",
-        "yarn.lock",
-      ];
-
-      expect(fse.readdirSync(path.resolve(cwd, testDir))).toStrictEqual(
-        expectDirs
-      );
-    });
-  });
+  nextPrompt(inputs);
 
-  describe("printing help message", () => {
-    test("--help flag works", async () => {
-      const { stdout } = await exe("node", [startFrontend, "--help"]);
-      expect(stdout.trim()).toBe(EXPECTED_HELP);
-    });
+  return new Promise((resolve) => cliProcess.on("exit", resolve));
+}
 
-    test("-h flag works", async () => {
-      const { stdout } = await exe("node", [startFrontend, "-h"]);
-      expect(stdout.trim()).toBe(EXPECTED_HELP);
-    });
+describe("start-frontend", () => {
+  beforeAll(() => {
+    // Initialize TEST_DIR before all tests
+    TEST_DIR =
+      // eslint-disable-next-line turbo/no-undeclared-env-vars
+      process.env.RUNNER_TEMP ||
+      execSync("mktemp -d -t my-test").toString("utf-8");
```

Comment:
Is there a reason why the return is being wrapped as a `utf-8` string? It doesn't seem like it's being used anywhere.